harvester/products/tests/seeding/test_anatomy_tool.py

```diff
@@ -0,0 +1,139 @@
+from datagrowth.configuration import register_defaults
+from django.test import TestCase
+
+from core.processors import HttpSeedingProcessor
+from core.tests.base import SeedExtractionTestCase
+from products.models import Set, ProductDocument
+from products.sources.anatomy_tool import AnatomyToolExtraction, SEEDING_PHASES
+from sources.factories.anatomy_tool.extraction import AnatomyToolOAIPMHFactory
+
+
+class TestAnatomyToolProductSeeding(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        register_defaults("global", {
+            "cache_only": True
+        })
+        AnatomyToolOAIPMHFactory.create_common_anatomy_tool_responses()
+
+    @classmethod
+    def tearDownClass(cls):
+        register_defaults("global", {
+            "cache_only": False
+        })
+        super().tearDownClass()
+
+    def setUp(self) -> None:
+        super().setUp()
+        self.set = Set.objects.create(identifier="srn")
+        self.processor = HttpSeedingProcessor(self.set, {
+            "phases": SEEDING_PHASES
+        })
+
+    def test_initial_seeding(self):
+        for batch in self.processor("anatomy_tool", "1970-01-01T00:00:00Z"):
+            self.assertIsInstance(batch, list)
+            for product in batch:
+                self.assertIsInstance(product, ProductDocument)
+                self.assertIsNotNone(product.identity)
+                self.assertTrue(product.properties)
+                if product.state == ProductDocument.States.ACTIVE:
+                    self.assertTrue(product.pending_at)
+                    self.assertIsNone(product.finished_at)
+                else:
+                    self.assertIsNone(product.pending_at)
+                    self.assertIsNotNone(product.finished_at)
+        self.assertEqual(self.set.documents.count(), 10)
+
+
+class TestAnatomyToolProductExtraction(SeedExtractionTestCase):
+
+    set = None
+    seeds = []
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        register_defaults("global", {
+            "cache_only": True
+        })
+        AnatomyToolOAIPMHFactory.create_common_anatomy_tool_responses()
+        cls.set = Set.objects.create(identifier="srn")
+        processor = HttpSeedingProcessor(cls.set, {
+            "phases": SEEDING_PHASES
+        })
+        cls.seeds = []
+        for batch in processor("anatomy_tool", "1970-01-01T00:00:00Z"):
+            cls.seeds += [doc.properties for doc in batch]
+
+    @classmethod
+    def tearDownClass(cls):
+        register_defaults("global", {
+            "cache_only": False
+        })
+        super().tearDownClass()
+
+    def test_get_complete_set(self):
```

Comment:
This can be deleted. These asserts are handles by TestAnatomyToolProductSeeding. So no need to repeat them here. New harvester tests have a bit more separation of what a TestCase is testing.